src/index.ts

```diff
@@ -36,6 +37,26 @@ export async function main() {
     )
     core.endGroup()
 
+    core.startGroup("Load json files into context")
+    const files = core.getInput("json-files-to-include")
+    if (files !== "") {
+      const filesJson = JSON.parse(files)
```

Comment:
This should be in a try/catch too, to avoid crashing this action on invalid inputs

Reply:
This should be typed, and ideally validated before usage.

```ts
/**
  * Metadata describing a file that contains the results of a test
  */
interface TestResultsFileMetadata {
	name: string
        path: string
}
```

Take a look at https://github.com/colinhacks/zod for data validation via parsing, it'll save us a lot of time later on

Reply:
Zod and try/catch implemented.